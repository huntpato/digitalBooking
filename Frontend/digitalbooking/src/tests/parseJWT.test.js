import React from 'react';
import parseJWT from '../components/UserLogin/parseJwt'
import { expect } from '@jest/globals';
import '@testing-library/jest-dom'

describe('Comprobar funcion parseJWT', ()=> {


    test('Verificar que la funcion se ejecuta correctamente', () => {
        const services = 'Bearer eyJhbGciOiJIUzI1NiJ9.eyJqdGkiOiJkaEpXVCIsInN1YiI6ImFndXNAZGguY29tIiwibmFtZSI6IkFndXN0aW5hIFBhc3F1YWxpcyIsImF1dGhvcml0aWVzIjpbIlVTRVIiXSwiaWF0IjoxNjM2NTU0Mjc2LCJleHAiOjE2MzY2NDA2NzZ9.xBvsAURnQm4QyaGalW81JcOx5MpDwu23xO_4lqJ_Cr8';
        expect(parseJWT(services)).toMatchSnapshot();;

})
})