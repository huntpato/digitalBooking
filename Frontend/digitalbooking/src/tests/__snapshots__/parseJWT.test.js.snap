// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Comprobar funcion parseJWT Verificar que la funcion se ejecuta correctamente 1`] = `
Object {
  "authorities": Array [
    "USER",
  ],
  "exp": 1636640676,
  "iat": 1636554276,
  "jti": "dhJWT",
  "name": "Agustina Pasqualis",
  "sub": "agus@dh.com",
}
`;
